cmd/node/main.go

```diff
@@ -717,6 +718,10 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	if err != nil {
 		return err
 	}
+
+	log.Trace("creating time cache for requested items components")
+	requestedItemsHandler := timecache.NewTimeCache(time.Duration(uint64(time.Millisecond) * nodesConfig.RoundDuration))
```

Comment:
request Items handler is actually a timeCache ?

Reply:
yep